### ISSUE TITLE
GUI fails to show segmentation when more than 1000 cells detected

### ISSUE THREAD

**Issue by giocard** [MouseLand/cellpose#428]
When the cells detected are more than 1000 you get the message
_ERROR: index 1001 is out of bounds for axis 0 with size 1001_
from
https://github.com/MouseLand/cellpose/blob/04b03d9adba988c1ea8fd2ed6c95f0fd8ce3f652/cellpose/gui/gui.py#L1316
Basically _cellcolors_ is always capped at 1000.
The message is repeated every time the MASKS ON checkbox is selected.

**Comment by yzlc080733** [comment]
Same issue here. I tried version 0.8.0 and 1.0.0.

I found a temporary solution:

1. open ~/anaconda3/envs/cellpose/lib/python3.7/site-packages/cellpose/gui/gui.py
2. find the line `self.colormap = (plt.get_cmap('gist_ncar')(np.linspace(0.0,.9,5000)) * 255).astype(np.uint8)`
3. there are three `1000` around this line, replace it with some larger number, like 5000 in my case.

This works for me. One possible drawback is that it may cause inconsistency in the colormaps between old setting (1000) and the new setting (5000). Not so harmful for me since currently I do not care about the absolute colors used ...  But still hope there can be a better solution, e.g., enlarging the colormap automatically according to the segmentation result.
